backend/db/ddl-14-concepts_with_counts_ungrouped.jinja.sql

```diff
@@ -17,5 +17,5 @@ SELECT DISTINCT
 FROM {{schema}}concept c
 LEFT JOIN {{schema}}deidentified_term_usage_by_domain_clamped tu ON c.concept_id = tu.concept_id);
 
-CREATE INDEX ccu_idx1 ON {{schema}}concepts_with_counts_ungrouped(concept_id);
---CREATE INDEX ccu_idx2 ON concepts_with_counts_ungrouped(concept_id);
\ No newline at end of file
+CREATE INDEX IF NOT EXISTS ccu_idx1 ON {{schema}}concepts_with_counts_ungrouped(concept_id);
```

Comment:
@hugwuoke Adding this will get around the error you were seeing. It was trying to create an index that already existed for some reason.